styles/components/Search.module.scss

```diff
@@ -0,0 +1,44 @@
+@import "../utils/mixins";
+
+.searchContainer {
+  background-color: var(--nypl-colors-ui-gray-x-light-cool);
+  padding: var(--nypl-space-l) 0 var(--nypl-space-xs);
+}
+
+.searchFormContainer {
+  @include maxWidth;
+
+  @include media(var(--nypl-breakpoint-xl)) {
+    > div {
+      width: 85%;
+    }
+  }
+
+  form {
+    .searchBarContainer {
```

Comment:
I don't think this rule does anything.

Reply:
Good catch, this was a holdover from a different approach I'd taken. Removed.